### PR TITLE
[TESTS] Get hosts parameters from container

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -116,6 +116,7 @@ tfp: vendor assets-amp
 	$(EXEC) $(CONSOLE) doctrine:database:create --if-not-exists --env=test_mysql || true
 	$(EXEC) $(CONSOLE) doctrine:schema:drop --force --env=test_mysql || true
 	$(EXEC) $(CONSOLE) doctrine:schema:create --env=test_mysql || true
+	$(EXEC) $(CONSOLE) doctrine:query:sql "ALTER TABLE groups CHANGE address_country address_country VARCHAR(2) DEFAULT NULL" --env=test_mysql || true
```
Comment (blazarecki) [inline Makefile]: Why ? Can you create a doctrine migration instead ?

File: tests/Mailjet/Message/EventNotificationMessageTest.php [hunk 2]
```diff
@@ -5,12 +5,11 @@
 use AppBundle\Entity\Adherent;
 use AppBundle\Mailjet\Message\EventNotificationMessage;
 use AppBundle\Mailjet\Message\MailjetMessageRecipient;
-use Tests\AppBundle\Config;
 
 class EventNotificationMessageTest extends AbstractEventMessageTest
 {
-    const SHOW_EVENT_URL = 'https://'.Config::APP_HOST.'/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon';
-    const ATTEND_EVENT_URL = 'https://'.Config::APP_HOST.'/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon/inscription';
+    const SHOW_EVENT_URL = 'https://enmarche.dev/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon';
```
Comment (blazarecki) [inline tests/Mailjet/Message/EventNotificationMessageTest.php]: can you use the parameter here please.

File: tests/Mailjet/Message/EventCancellationMessageTest.php [hunk 3]
```diff
@@ -6,11 +6,10 @@
 use AppBundle\Mailjet\Message\EventCancellationMessage;
 use AppBundle\Mailjet\Message\EventNotificationMessage;
 use AppBundle\Mailjet\Message\MailjetMessageRecipient;
-use Tests\AppBundle\Config;
 
 class EventCancellationMessageTest extends AbstractEventMessageTest
 {
-    const SEARCH_EVENTS_URL = 'https://'.Config::APP_HOST.'/evenements';
+    const SEARCH_EVENTS_URL = 'https://enmarche.dev/evenements';
```
Comment (blazarecki) [inline tests/Mailjet/Message/EventCancellationMessageTest.php]: Same here

File: tests/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessageTest.php [hunk 4]
```diff
@@ -2,12 +2,11 @@
 
 namespace AppBundle\Mailjet\Message;
 
-use Tests\AppBundle\Config;
 use Tests\AppBundle\Mailjet\Message\AbstractEventMessageTest;
 
 class CommitteeCitizenInitiativeOrganizerNotificationMessageTest extends AbstractEventMessageTest
 {
-    const CONTACT_ADHERENT_URL = 'https://'.Config::APP_HOST.'/espace-adherent/contacter/a9fc8d48-6f57-4d89-ae73-50b3f9b586f4?from=committee&id=464d4c23-cf4c-4d3a-8674-a43910da6419';
+    const CONTACT_ADHERENT_URL = 'https://enmarche.dev/espace-adherent/contacter/a9fc8d48-6f57-4d89-ae73-50b3f9b586f4?from=committee&id=464d4c23-cf4c-4d3a-8674-a43910da6419';
```
Comment (blazarecki) [inline tests/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessageTest.php]: And here

File: tests/Mailjet/Message/EventNotificationMessageTest.php (hunk 2 above)
Comment (Remg) [reply tests/Mailjet/Message/EventNotificationMessageTest.php]: its a unit test. not a functional test, so no container here

File: tests/Mailjet/Message/EventCancellationMessageTest.php (hunk 3 above)
Comment (Remg) [reply tests/Mailjet/Message/EventCancellationMessageTest.php]: same as above

File: tests/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessageTest.php (hunk 4 above)
Comment (Remg) [reply tests/Mailjet/Message/CommitteeCitizenInitiativeOrganizerNotificationMessageTest.php]: same as above

File: Makefile (hunk 1 above)
Comment (Remg) [reply Makefile]: tests via makefile does not run migrations:
`doctrine:schema:create --env=test_mysql`

Comment (blazarecki) [reply Makefile]: So the schema is not sync with the model.

Comment (Remg) [reply Makefile]: indeed. another PR will fix this

**Pull request merged**